data/mods/more_classes_scenarios/cs_classes.json

```diff
@@ -260,5 +260,219 @@
+            "points": -1,
+            "items": {
+                "both": [
+                    "tshirt",
+                    "socks",
+                    "lowtops",
+                    "shorts"
+                ],
+                "male": [
+                    "briefs"
+                ],
+                "female": [
+                    "panties"
+                ]
+            }
+    },
+    {
+            "type": "profession",
+            "ident": "elementary_student",
+            "name": "Elementary Student",
+            "description": "You're just a kid, and now the world has turned into something out of one of your bad dreams.  The grown-ups you relied on are all dead--or undead--now.  What are you going to do?",
+            "points": 0,
+            "items": {
+                "both": [
+                    "tshirt",
+                    "socks",
+                    "slingpack",
+                    "sneakers",
+                    "jeans",
+                    "jacket_light",
+                    "knit_scarf",
+                    "juice",
+                    "sandwich_pbj",
+                    "fairy_tales"
+                ],
+                "male": [
+                    "briefs"
+                ],
+                "female": [
+                    "panties"
+                ]
+            }
+    },
+    {
+        "type": "profession",
+        "ident": "goalie",
+        "name": "Hockey Player",
+        "description": "You were a minor-league hockey goalie before the rest of your team became zombies.  It's just you and your hockey equipment versus the undead, but at least you can cross-check them now.",
+        "points": 0,
+        "traits": [
+            "PROF_SKATER"
+        ],
+        "items": {
+            "both": [
+                "mask_hockey",
+                "hockey_stick",
+                "jersey",
+                "pants",
+                "socks",
+                "sneakers",
+                "mouthpiece",
+                "puck"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "sports_bra",
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "batter",
+        "name": "Baseball Player",
+        "description": "You were a batter on a local minor league team before the cataclysm.  You escaped with your equipment, but how long can you survive until your innings are up?",
+        "points": 0,
+        "items": {
+            "both": [
+                "bat",
+                "helmet_ball",
+                "dress_shirt",
+                "pants",
+                "socks",
+                "cleats",
+                "mouthpiece",
+                "baseball"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "sports_bra",
+                "panties"
+            ]
+        }
```

Comment:
Bat is a good start, but 1 point in melee and/or bashing would make it more sensible.